hardhat.network.ts

```diff
@@ -92,7 +92,7 @@ if (mnemonic) {
 
 if (infuraApiKey && mnemonic) {
   networks.optimismkovan = {
-    url: `https://optimism-kovan.infura.io/v3/${infuraApiKey}`,
+    url: process.env.ALCHEMY_URL,
```

Comment:
Could you instead call this "OPTIMISM_KOVAN_RPC_URL" or something?

Reply:
Done: https://github.com/pooltogether/v4-testnet/pull/36/commits/5c5fb57e35092dad841799be64a7f4357ebec038